backend/dataall/core/environment/aws/ec2_client.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from dataall.base.aws.sts import SessionHelper
+from botocore.exceptions import ClientError
+
+log = logging.getLogger(__name__)
+
+
+class EC2:
+
+    @staticmethod
+    def get_client(account_id: str, region: str, role=None):
+        session = SessionHelper.remote_session(accountid=account_id, role=role)
+        return session.client('ec2', region_name=region)
+
+    @staticmethod
+    def check_default_vpc_exists(AwsAccountId: str, region: str, role=None):
```

Comment:
Since this function is used in `environments` and in `mlstudio` we should move the ec2 class to `base` to indicate that it is a shared library

Reply:
Yes we definitely should! - I will move